### ISSUE TITLE
ssh_config percent_expand: unknown key %C

### ISSUE THREAD

**Issue by ritzmann** [Xpra-org/xpra#3440]
**Describe the bug**
`xpra shadow` fails to start on my Mac with my SSH configuration. The error message is:
```
xpra initialization error:
 connection failed: ssh_config percent_expand: unknown key %C
```

**To Reproduce**
1. server command: Regular xsession
2. client command that is failing: `xpra shadow ssh://user@machine/`
3. client .ssh/config:
```
Host *
    ControlMaster auto
    ControlPath /tmp/ssh-cm-%C
    ControlPersist 30m
```

The connection succeeds when I comment out `ControlPath`. `%C` is commonly used with ControlPath.

**System Information (please complete the following information):**
 - Server OS: Debian 11
 - Client OS: macOS 11.6.2
 - Xpra Server Version 3.0.13
 - Xpra Client Version 4.3.1 (installed via Homebrew)

**Additional context**

**Comment by totaam** [comment]
This is a limitation of paramiko so there's nothing we can do in xpra:
https://github.com/paramiko/paramiko/blob/88f35a537428e430f7f26eee8026715e357b55d6/paramiko/config.py#L61-L69

But you can workaround it by using the openssh backend instead: `--ssh=ssh -x`
(but you will lose niceties like GUI password prompts, etc)

**Comment by ritzmann** [comment]
All clear. I opened a feature request upstream: https://github.com/paramiko/paramiko/issues/1976

**Comment by totaam** [comment]
This fix is included in paramiko 2.10
